api-consumption/src/main/java/org/zalando/nakadi/service/subscription/state/StreamingState.java

```diff
@@ -426,33 +427,34 @@ private int getBatchLimitEvents() {
         return getParameters().batchLimitEvents;
     }
 
-    private Optional<String> getDebugMessage(final Map.Entry<EventTypePartition, PartitionData> entry) {
-        final PartitionData partitionData = entry.getValue();
-        final long skippedEventsCount = partitionData.getSkippedEventsCount();
-        if (skippedEventsCount > 0) {
-            partitionData.resetSkippedEventsCount();
-            return Optional.of(String.format(
-                    "Skipped events due to retention time passed, count: %d", skippedEventsCount));
-        }
+    private Optional<String> makeDebugMessage(final PartitionData partitionData) {
+        final StringBuilder sb = new StringBuilder();
 
         if (batchesSent == 0) {
-            final StringBuilder sb = new StringBuilder();
             sb.append("Stream started");
 
-            final String failedCommitsPartitions = Arrays.stream(getZk().getTopology().getPartitions())
-                    .filter(p -> p.getFailedCommitsCount() > 0 || p.isLookingForDeadLetter())
-                    .map(Partition::toFailedCommitString)
-                    .collect(Collectors.joining(", "));
+            final long skippedEventsCount = partitionData.getSkippedEventsCount();
+            if (skippedEventsCount > 0) {
+                partitionData.resetSkippedEventsCount();
+                sb.append("; skipped events due to retention time passed, count: ")
+                        .append(skippedEventsCount);
+            }
+        }
 
-            if (failedCommitsPartitions != null && !failedCommitsPartitions.isEmpty()) {
-                sb.append(". Failed commits: ").append(failedCommitsPartitions);
-                LOG.info("Failed commits: {}", failedCommitsPartitions);
+        if (failedCommitsDebugStringToFlush != null && !failedCommitsDebugStringToFlush.isEmpty()) {
+            if (sb.length() != 0) {
+                sb.append("; ");
             }
+            sb.append("failed commits tracking: ")
```

Comment:
I would explicitly mention what those failed commits are about
```suggestion
            sb.append("Auto DLQ failed commits tracking: ")
```